### PR TITLE
Add docker

### PR DESCRIPTION
**Pull request by pkowalsk1**

### CODE REVIEW

File: .github/workflows/build-docker-image.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        description: 'tag that the image will be built with'
+        required: true
+        default: 'noetic'
+      branch:
+        description: 'branch that will be used to build image'
+        required: true
+        default: 'main'
+  
+jobs:
+  build_velodyne_ros:
```
Comment (KmakD) [inline .github/workflows/build-docker-image.yaml]:
```suggestion
  build_nmea_gps_ros:
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+WORKDIR /ros_ws
+
+RUN apt-get update  && \
+    apt-get install -y \
+        git \
+        python3-pip && \
+    pip3 install \
+        rosdep && \
+    git clone -b decode-udp-line https://github.com/ros-drivers/nmea_navsat_driver.git src/nmea_navsat_driver && \
```
Comment (KmakD) [inline Dockerfile:13]: Is there a particular reason we use this branch?

**Review by KmakD — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (pkowalsk1) [reply Dockerfile:13]: Regarding the [documentation](https://husarion.com/manuals/panther/#software-guide), it is recommended to use this branch. This is the only properly working version of this software.

Comment (KmakD) [reply Dockerfile:13]: ok

**Review by KmakD — APPROVED** [review]

**Pull request merged**